draft-ietf-rats-architecture.md

```diff
@@ -787,16 +787,25 @@ in order to bootstrap the sequence.
 The Verifier trusts (or more specifically, the Verifier's security
 policy is written in a way that configures the Verifier to trust) a
 manufacturer, or the manufacturer's hardware, so as to be able to
-appraise the trustworthiness of that manufacturer's devices.  In solutions
-with weaker security, a Verifier might be configured to implicitly
-trust firmware or even software (e.g., a hypervisor).  That is, it
-might appraise the trustworthiness of an application component, operating
+appraise the trustworthiness of that manufacturer's devices.
+
+In solutions with weaker security, a Verifier might be configured to implicitly
+trust firmware or even software (e.g., a hypervisor).
+That is, it might appraise the trustworthiness of an application component, operating
 system component, or service under the assumption that information
-provided about it by the lower-layer hypervisor or firmware is true.
+provided about it by the lower-layer firmware or software is true.
 A stronger level of assurance of security comes when information can be vouched
 for by hardware or by ROM code, especially if such hardware is
-physically resistant to hardware tampering.  The component that is
-implicitly trusted is often referred to as a Root of Trust.
+physically resistant to hardware tampering.
+The component that is implicitly trusted is often referred to as a Root of Trust.
```

Comment:
```suggestion
In most cases, components that have to be vouched for via Endorsement because no Evidence can be created about them are referred to as a roots of trust.
```